Libplanet.Tests/KeyStore/Ciphers/Aes128CtrTest.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using System.Collections.Immutable;
+using System.Text.Json;
+using Libplanet.KeyStore;
+using Libplanet.KeyStore.Ciphers;
+using Libplanet.KeyStore.Kdfs;
+using Org.BouncyCastle.Crypto.Digests;
+using Xunit;
+
+namespace Libplanet.Tests.KeyStore.Ciphers
+{
+    public class Aes128CtrTest : CipherTest<Aes128Ctr>
+    {
+        private Aes128Ctr _cipher;
+
+        public Aes128CtrTest()
+        {
+            var random = new Random();
+            var buffer = new byte[16];
+            random.NextBytes(buffer);
+            _cipher = new Aes128Ctr(buffer.ToImmutableArray());
+        }
+
+        public override Aes128Ctr Cipher => _cipher;
+
+        [Fact]
+        public void Constructor()
+        {
+            Assert.Throws<ArgumentException>(() =>
+                new Aes128Ctr(new byte[0].ToImmutableArray())
+            );
+            var random = new Random();
+            var buffer = new byte[17];
+            random.NextBytes(buffer);
+            Assert.Throws<ArgumentException>(() =>
+                new Aes128Ctr(buffer.ToImmutableArray())
+            );
+        }
+
+        [Fact]
+        public void FromJson()
+        {
+            var options = new JsonDocumentOptions
+            {
+                AllowTrailingCommas = true,
+                CommentHandling = JsonCommentHandling.Skip,
+            };
+
+            Aes128Ctr Load(string json)
+            {
+                using (JsonDocument doc = JsonDocument.Parse(json, options))
+                {
+                    return (Aes128Ctr)Aes128Ctr.FromJson(doc.RootElement);
+                }
+            }
+
+            Aes128Ctr cipher = Load(@"{
+                ""iv"": ""bc7f2ca23bfee0dd9725228ab2b0d98a"",
```

Comment:
How about using `JObject` instead of verbatim string literals?

Reply:
We're not using Newtonsoft.Json, but [`System.Text.Json`](https://docs.microsoft.com/en-us/dotnet/api/system.text.json) which lacks `JObject`.

Reply:
My thought was to use both `Newtonsoft.Json` and `System.Text.Json` in the test case. But on second thoughts, string literals look better than `JObject`.